### ISSUE TITLE
0.12 news post dead link to CompressedImageSaver

### ISSUE THREAD

**Issue by GitGhillie** [bevyengine/bevy-website#1049]
Dunno if this is the right spot for this, but I was looking at https://bevyengine.org/news/bevy-0-12/#compressedimagesaver and when I pressed the link to CompressedImageSaver (https://docs.rs/bevy/0.12.0/bevy/render/texture/struct.CompressedImageSaver.html) I got "The requested resource does not exist". And I can't seem to find what happened to CompressedImageSaver.

**Comment by TrialDragon** [comment]
It appears CompressedImageSaver does exist, but isn't being recorded in the docs at all? That link is supposed to be valid.

**Comment by alice-i-cecile** [comment]
I bet it's not included in the default features, so it's not showing up on docs.rs.

**Comment by TrialDragon** [comment]
It would appear so. It's under `basis-universal` which isn't a default feature. Is there any way to make these features visible for users? Like something to enable client-side on docs.rs, or maybe something we have to do on our end?

**Comment by alice-i-cecile** [comment]
https://github.com/google/mundane/issues/21

Looks like this is the field we need. IIRC dev-docs is already built with all features: see https://dev-docs.bevyengine.org/bevy/render/texture/struct.CompressedImageSaver.html

**Comment by TrialDragon** [comment]
Will this be solved by https://github.com/bevyengine/bevy/pull/12366 when the next Bevy version releases on crates.io?

**Comment by alice-i-cecile** [comment]
No, it won't: that will only resolve similar problems for future releases. I think that this is probably not worth spending more effort to fix for an old news post though.
